packages/ui5-task-transpile/README.md

```diff
@@ -18,10 +18,11 @@ npm install ui5-task-transpile --save-dev
 
 - excludePatterns: `String<Array>`  
   array of paths inside `$yourapp/webapp/` to exclude from live transpilation,  
-array of paths inside `$yourapp/webapp/` to exclude from live transpilation,
-  array of paths inside `$yourapp/webapp/` to exclude from live transpilation,  
   e.g. 3-rd party libs in `lib/*`
 
+- transpileAsync: `true|false`
```

Comment:
same thing as in the `README.md` for livetranspiling: please add 3 spaces for a linkbreak